src/libraries/System.Net.Sockets/tests/FunctionalTests/ArgumentValidationTests.cs

```diff
@@ -794,18 +794,20 @@ public async Task Socket_ConnectAsync_IPv6AddressAsStringHost_Supported()
 
         [Theory]
         [PlatformSpecific(TestPlatforms.AnyUnix)]  // API throws PNSE on Unix
-        [InlineData(0)]
-        [InlineData(1)]
-        public void Connect_ConnectTwice_NotSupported(int invalidatingAction)
+        [InlineData(false)]
+        [InlineData(true)]
+        public void Connect_ConnectTwice_NotSupported(bool invalidatingAction)
```

Comment:
Can we leave the method signature unchganged and just do:

```C#
if (PlatformDetection.IsRiscV64Ubuntu && invalidatingAction == 1)
{
  return;
}
```
?

Same for `ConnectAsync_ConnectTwice_NotSupported`.